src/ch00-07-constructor.md

```diff
@@ -0,0 +1,18 @@
+# Constructor
+
+A blockchain can be seen as a data base, made of states that corresponds to the data inside the blockchain at a certain time. Moving from a state to another is dicted by the functions inside the smart contract. But we still need to have a starting state. 
+Normally, starting state is default values ( 0 for felt252, empty for mappings), but if you want, you can chose non default values as a starting point : this is the role of a constructor. 
```

Comment:
This introduction feels out of context here. I find it confusing and would prefer that we stay focused on constructors and remove it. The text below is sufficient

Reply:
I was proud of that :p